client/ivy.xml

```diff
@@ -10,7 +10,7 @@
   <dependency org="jaxen" name="jaxen" rev="1.1-beta-10"/>
   <dependency org="dom4j" name="dom4j" rev="1.5.2" />
   <dependency org="log4j" name="log4j" rev="1.2.16" />
-  <dependency org="com.google.guava" name="guava" rev="13.0.1" />
+  <dependency org="com.google.guava" name="guava" rev="23.0" />
```

Comment:
I assume that this commit does 2 things:
* updates the dependency in ivy.xml
* uses MoreObjects instead of Objects

In order to save reviewers time scanning through screeds of changes that the compiler is better at validating,
please change the commit comment with useful information.

Reply:
Done

Reply:
"and few other changes" 👎 So, my assumption was incorrect.  Sorry, but you will need to find other reviewers.